espnet2/asr/frontend/enh_frontend.py

```diff
@@ -0,0 +1,83 @@
+from typing import Dict
```

Comment:
By considering that `enh` can be used as a stand-alone tool, we may think of putting it in `espnet2/enh`? I want to ask @kamo-naoyuki's opinion about the design.
Of course, we strongly assume this function as a front-end processing for ASR, the current design is fine.

Reply:
I agree.

Reply:
I have moved it to `espnet2/enh`, the name of `frontend` has been removed.